src/components/ProfileForm/components/ProfileForm/ProfileForm.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+import { useState } from "react";
+import ProfileFormStep0 from "../ProfileFormStep0";
+import ProfileFormStep1 from "../ProfileFormStep1";
+import ProfileFormStep2 from "../ProfileFormStep2";
+import Button from "../../../Button-Link/Button/Button";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faAngleDoubleLeft,
+  faAngleDoubleRight,
+} from "@fortawesome/free-solid-svg-icons";
+import ProfileFormStep3 from "../ProfileFormStep3";
+import "./ProfileForm.css";
+
+const ProfileForm = () => {
+  const [step, setStep] = useState<number>(0);
+
+  const advanceStep = (e: any) => {
+    e.preventDefault();
+    if (step < STEP_COMPONENTS.length - 1) {
+      setStep(step + 1);
+    }
+  };
+
+  const decreaseStep = (e: any) => {
```

Comment:
Also here. Let's not use `any`.

Reply:
Now that I took a second look at this code, there's no need for this `e` and the `preventDefault` code in this one, because it's not a submit action. Just a normal click action.